iogt/static/css/modules/_m-menu.scss

```diff
@@ -10,17 +10,11 @@
     font-size: $font-small;
```

Comment:
On line 4 there's a font-size that isn't a variable; I can't reference it in here for some reason :/